envs/dlbclass/dlbclass.yaml

```diff
@@ -0,0 +1,16 @@
+name: dlbclass-test
```

Comment:
just wondering if this is exported using --from-history flag? Maybe we should consider switch to that one so the envs are easier to rebuild over time 😅

Reply:
Yes this is the `--from-history` version - very minimal and much less than what you'd see if I'd exported everything!

Reply:
Thanks for confirming! This is great!